docs/guides/serving.md

```diff
@@ -51,6 +51,8 @@ if __name__ == "__main__":
 
 You can also view a complete example [here](https://github.com/floydhub/fast-style-transfer/blob/master/app.py).
 
+!!! important "Pay attention to Race Conditions!"
+    The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of race condition: remember to lock the disk resources every time you will need to write and read from them. 
```

Comment:
Can you provide an example of how to lock the disk resources?

Reply:
we should just avoid doing it, it's not a good idea to write to disk for serving anyway, it will slow down the start up time and I can't think of a good use-case for this.

Reply:
Yeah, this sounds good 👍